### ISSUE TITLE
Empty scroll bar in code samples

### ISSUE THREAD

**Issue by zachfedor** [NoDivide/astrum#69]
#### Tell us about the issue you are experiencing

Code sample styles include `overflow-x: scroll;` which shows unnecessary scroll bar.

More info in PR #68

---

Platform: OSX
OS Version: 10.11.6
Node Version: 5.3.0
NPM Version: 3.5.2
Astrum Version: 1.8.0

**Comment by RyanHavoc** [comment]
@zachfedor I think you may have inadvertently fixed a bug. We never intended for the scrollbar to be shown and they don't appear in Chrome for OSX for me. What browser are you using?

**Comment by zachfedor** [comment]
@RyanHavoc I just figured out the problem. I'm also using Chrome but there is a setting in System Preferences under the General tab to control the appearance of scroll bars. I think OSX's default is to only show them while scrolling, but it still keeps the 17px (or whatever it is) space for it even if it doesn't show anything. I've been burned by this behavior before, so I set it to always show the scroll bars if they're there. I think it mirrors other platforms a bit better, but I could be wrong.

If this is just a system prefs thing, I'm fine with just changing my personal settings instead of changing default behavior. Another option is to change the default CSS but remove the user option in `data.json` so the theme config doesn't get bloated.

Now that I think of it, is it worthwhile adding an easier way for user's to add their own styles to Astrum so that they aren't changing `app/css/styles.*` which, I imagine, gets blown away on `astrum update`?
